root/static/scripts/guess-case/MB/GuessCase/Main.js

```diff
@@ -119,9 +119,10 @@ const flags = require('../../flags');
         sortname: guess("Work", "guessSortName")
     };
 
-    // Series doesn't have it's own handler, and just uses the work handler
+    // Series and Event doesn't have it's own handler, and just uses the work handler
```

Comment:
Yeah, I guess you could move “handler” to the next line, although we don’t really have a hard limit.

While you are touching the comment, you could also correct “it's own handler” into “its own handler”.